device-trees/system-device-tree.dts

```diff
@@ -54,6 +56,30 @@
 			};
 		};
 	};
+	ps_ipi_1 {
```

Comment:
I recall feedback that if there's a reg property in the node, the name should include the unit address, i.e. https://elinux.org/Device_Tree_Usage#Node_Names

If so, should we add some @<unit addresses> to these ?

Reply:
added unit address to  name in later commit.